### ISSUE TITLE
Fix issues reported by lighthouse

### ISSUE THREAD

**Issue by mvpoyatt** [polymerdao/ibc-explorer#44]
Description
------------
-  On public explorer, go to dev tools and run lighthouse for desktop. SEO and Best Practices are both 100%, but accessibility and performance are both 90-97%. These are good numbers but the test results also provide areas for improvement. Implement these changes where possible to improve score.

Reasoning
----------
- Better UX and page ranking

Acceptance Criteria
--------------------
1. 95+ for performance and accessibility. Why not 100? Sometime the desired styling choices lower accessibility score, for example if contrast isn't high enough between background and text because a button is disabled. For performance higher is always better but things outside of UI code can impact this.
2. Once this is done create a new issue to do the same for mobile.
